### PR TITLE
graph & roe value modification

### PR DESCRIPTION
**Pull request by ravirathod98**

### CODE REVIEW

File: Management/views.py [hunk 1]
```diff
@@ -32317,10 +32317,11 @@ def view_detailed_result(request):
             people_daily_commute = people.filter(subcategory='Daily Commute')
 
             laptop = get_direct_impact_data_for_current_project.filter(subcategory='Laptop')
-            pc = get_direct_impact_data_for_current_project.filter(subcategory='High-performance PC')
+            # pc = get_direct_impact_data_for_current_project.filter(subcategory='High-performance PC')
+            pc = get_direct_impact_data_for_current_project.filter(subcategory='Desktop')
             tablet = get_direct_impact_data_for_current_project.filter(subcategory='Tablet')
             monitor = get_direct_impact_data_for_current_project.filter(subcategory='Screen/Monitor')
-            telephone = get_direct_impact_data_for_current_project.filter(subcategory='Smartphone/Telephone')
```
Comment (Vishal-Capgemini2121) [inline Management/views.py:32323]: Did you check the category in DB for Telephone?

**Review by Vishal-Capgemini2121 — COMMENTED** [review]
1 Minor comment otherwise LGTM.

File: Management/views.py (hunk 1 above)
Comment (ravirathod98) [reply Management/views.py:32323]: Yes, category in DB for Telephone is "Telephone".

Comment (Vishal-Capgemini2121) [reply Management/views.py:32323]: Ok great.

**Pull request merged**